packages/gatsby/package.json

```diff
@@ -48,7 +48,7 @@
     "domready": "^1.0.8",
     "dotenv": "^4.0.0",
     "eslint": "^4.19.1",
-    "eslint-config-react-app": "3.0.0-next.66cc7a90",
+    "eslint-config-react-app": "^3.0.0-next.66cc7a90",
```

Comment:
May as well just call this `^3.0.0` right?

Reply:
(although they're semantically equivalent, it's probably fine to move to a major version)